README.rst

```diff
@@ -75,18 +77,49 @@ Please include all of the following information:
 
 - The operating system and version (e.g. RedHat Enterprise Linux 6.4, OSX 10.9.5, ...)
 
-Documentation
-=============
+Security Vulnerabilities
+------------------------
 
-Motor's documentation is on ReadTheDocs_.
+If you’ve identified a security vulnerability in a driver or any other
+MongoDB project, please report it according to the `instructions here
+<http://docs.mongodb.org/manual/tutorial/create-a-vulnerability-report>`_.
 
-Build the documentation with Python 3.5. Install sphinx, Tornado, and aiohttp,
-and do ``cd doc; make html``.
+Installation
+============
+
+Motor can be installed with `pip <http://pypi.python.org/pypi/pip>`_::
+
+  $ pip install motor
+
+You can also download the project source and do::
+
+  $ python setup.py install
```

Comment:
Let's remove this option. Or replace it with `python3 -m pip install .`

Reply:
Removed it from the README. Added to the Installation section of docs.